apps/designer/app/canvas/shared/styles.ts

```diff
@@ -190,6 +216,14 @@ const useUpdateStyle = () => {
   });
 };
 
+const useUpdateTokens = () => {
+  useSubscribe("updateTokens", (updatedTokens) => {
+    store.createTransaction([designTokensContainer], (tokens) => {
+      tokens.splice(0, tokens.length, ...updatedTokens);
```

Comment:
This means patch objects will always contain full rewrite of all tokens, right? Not sure if this matters now, but probably will cause problems with collaboration for example.

Reply:
I think I made a mistake by wanting to update all tokens instead of one